### ISSUE TITLE
feat: add color palettes for users with Color Vision Deficiency (CVD)

### ISSUE THREAD

**Issue by steven-aj** [steven-aj/steven-aj.github.io#29]
The site doesn't currently support users with CVD and fails accessibility benchmarks for color contrast.

This feature will add support for: 

- **Protanopia** – red-blind
- **Deuteranopia** – green-blind
- **Tritanopia** – blue-blind
- **Achromatopsia** – total color blindness (no color perception)

As well as:

- contrast enhancements that pass accessibility standards
